admission-server/README.md

```diff
@@ -0,0 +1,38 @@
+# Example Implementation of the Admission Server
+
+The Admission Server is an optional component of the Cosmo Platform to validate the deployment of the Router configuration before it is applied to the Router.
+
+This example implementation is a simple web server that implemented the `/validate-config` hook to sign and validates the Router configuration. The server is implemented in TypeScript and uses the Hono web framework.
+
+```bash
+pnpm install
+pnpm run dev
+```
+
+Open the browser and navigate to the following URL:
+```
+open http://localhost:3009
+```
+
+# Demo
+
+1. Create a graph with an admission webhook url:
+
+```bash
+wgc federated-graph create mygraph -r http://127.0.0.1:3001/graphql --admission-webhook-url http://localhost:3009 --label-matcher=team=A,team=B
+```
+
+2. Publish a subgraph
+```bash
+wgc subgraph publish employees --schema ../demo/pkg/subgraphs/employees/subgraph/schema.graphqls --labels team=A
+```
+
+3. Start the router with the same signing key as the admission server
+```yaml
+version: '1'
+
+graph: 
+  sign_key: 'sign_key'
```

Comment:
how about renaming to "config_signature_key" and adding "config_signature_algorithm" enum with a default value of what you've implemented to keep the alg flexible.

Reply:
That would require to match the right algorithm in the admission handler. I'd like to be opinionated here to make it simpler. HMAC256 is a reasonable default which is used across the industry. Let's wait for feedback also.